src/vivarium_gates_child_iv_iron/model_specifications/iv_iron_child.yaml

```diff
@@ -53,8 +53,10 @@ components:
 configuration:
     input_data:
         input_draw_number: 0
-        artifact_path: '/ihme/costeffectiveness/artifacts/vivarium_gates_child_iv_iron/south_asia.hdf'
-        fertility_input_data_path: '/ihme/costeffectiveness/results/vivarium_gates_iv_iron/v8.2_child_output_only/south_asia/2022_06_16_12_36_52/child_data/'
+        # Artifact should be defined at runtime using -i flag (available in src/vivarium_gates_iv_iron/artifacts)
+        #artifact_path: '/ihme/costeffectiveness/artifacts/vivarium_gates_child_iv_iron/south_asia.hdf'
+        # Use child_data folder created by IV iron maternal model
```

Comment:
Is `-i` a required argument? I don't think it is, and the "Artifact *should*" bit seems wrong. IIRC from past archivals, we simply use the relative path from the model spec for a given artifact.

Reply:
Having a relative path is just fine here, but I wasn't sure if that would work. If you test it and it works that's perfect.